e2e/main.e2e.spec.ts

```diff
@@ -53,32 +53,39 @@ describe('Rancher Desktop', () => {
   });
 
   it('should display welcome message in general tab', async() => {
-    const generalPage: GeneralPage = await navBarPage.getGeneralPage();
+    const generalPage = await navBarPage.getGeneralPage();
 
-    expect(await generalPage.getTitle()).toBe('Welcome to Rancher Desktop');
+    expect(generalPage).not.toBeNull();
+    expect(await generalPage?.getTitle()).toBe('Welcome to Rancher Desktop');
   });
 
   it('should switch to Kubernetes Settings tab', async() => {
-    const kubernetesPage: KubernetesPage = await navBarPage.getKubernetesPage();
+    const kubernetesPage = await navBarPage.getKubernetesPage();
 
-    expect(await kubernetesPage.getResetKubernetesButtonText()).toBe('Reset Kubernetes');
+    expect(kubernetesPage).not.toBeNull();
+    expect(await kubernetesPage?.getResetKubernetesButtonText()).toBe('Reset Kubernetes');
   });
 
   it('should switch to Port Forwarding tab', async() => {
-    const portForwardingPage: PortForwardingPage = await navBarPage.getPortForwardingPage();
+    const portForwardingPage = await navBarPage.getPortForwardingPage();
 
-    os.platform().startsWith('win') ? expect(portForwardingPage).not.toBeNull() : expect(portForwardingPage).toBeNull();
+    if (os.platform().startsWith('win')) {
```

Comment:
I would test for darwin specifically, in case we add linux support and port-forwarding works there.

Reply:
Make sense, though I was just transforming the existing code. We can fix this when the check is no longer accurate.

Reply:
fine